### PR TITLE
Tezos dart library v1.0.0

### PR DESCRIPTION
**Pull request by Rakshak1344**
v1.0.0

### CODE REVIEW

**Comment by jvtezsure** [comment]
Resolve confilcts.

**Review by jvtezsure — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# [0.0.1]
```
Comment (jvtezsure) [inline CHANGELOG.md]: Let's start with 1.0.0
pls change it to all the different places you've mentioned 0.0.1

**Review by jvtezsure — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Rakshak1344) [reply CHANGELOG.md]: version changed.

**Review by Rakshak1344 — COMMENTED** [review]
Did changes, please review it.

**Review by jvtezsure — APPROVED** [review]

**Pull request merged**